development/package.json

```diff
@@ -0,0 +1,10 @@
+{
+  "name": "@homarr/dev-db",
```

Comment:
Name is misleading, the compose file *could* contain things besides a DB

Reply:
Fixed it ;)